pulp_container/tests/functional/api/test_push_signatures.py

```diff
@@ -0,0 +1,172 @@
+# coding=utf-8
+"""Tests that verify that images can be pushed to Pulp."""
+import json
+import os
+import shutil
+import re
+import requests
+import subprocess
+import unittest
+
+from urllib.parse import urlparse
+
+from pulp_smash import cli, config
+from pulp_smash.pulp3.bindings import (
+    delete_orphans,
+    monitor_task,
+    PulpTestCase,
+)
+from pulp_container.tests.functional.api import rbac_base
+from pulp_container.tests.functional.constants import REGISTRY_V2_REPO_PULP
+from pulp_container.tests.functional.utils import (
+    gen_container_client,
+    get_auth_for_url,
+)
+
+from pulpcore.client.pulp_container import (
+    DistributionsContainerApi,
+    RepositoriesContainerPushApi,
+    ContentSignaturesApi,
+    ContentManifestsApi,
+)
+
+
+class PushSignatureRepoTestCase(PulpTestCase, rbac_base.BaseRegistryTest):
```

Comment:
I would rather prefer new tests to be written with the new pytest style.

Reply:
Okay, I am waiting for your PR (https://github.com/pulp/pulp_container/pull/632) to be merged. I see that you added fixture `local_storage` which I would like to use.

Reply:
I undrafted it so we can get that work unblocked.